Doc/library/turtle.rst

```diff
@@ -749,9 +749,10 @@ Turtle motion
    the turtle; *extent* -- an angle -- determines which part of the circle
    is drawn.  If *extent* is not given, draw the entire circle.  If *extent*
    is not a full circle, one endpoint of the arc is the current pen
-   position.  Draw the arc in counterclockwise direction if *radius* is
-   positive, otherwise in clockwise direction.  Finally the direction of the
-   turtle is changed by the amount of *extent*.
+   position. Draw the arc in counterclockwise direction if both *radius*
+   and *extent* are positive or both are negative; otherwise, draw in clockwise
+   direction.  Finally the direction of the turtle is changed by the amount of
+   *extent*.
```

Comment:
When *radius* and *extent* are positive, the arc is counterclockwise. When *radius* and *extent* are both negative, one traverses the standard counterclockwise arc **backwards**. I wonder if this could be communicated better here?